catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -172,6 +182,12 @@ public QueryRequest createQueryRequest(String localSource, FilterBuilder filterB
           .put(ADDITIONAL_SORT_BYS, sortBys.subList(1, sortBys.size()).toArray(new SortBy[0]));
     }
 
+    queryRequest.getProperties().put("requestId", id);
+
+    if (StringUtils.isNotEmpty(batchId)) {
+      queryRequest.getProperties().put("batchId", batchId);
```

Comment:
Where is this value ever read from? Do we need any sort of validation around this field to ensure that it is unique and the correct format? What happens if these aren't unique? You can't rely on the UI to send an actual unique value to you without validating that it is unique on the back end.

Reply:
Correlation ID is the more common name for this.